apps/storybook.namekit.io/stories/Namekit/CurrencySymbol.stories.tsx

```diff
@@ -1,5 +1,6 @@
 import { Currency } from "@namehash/ens-utils";
-import { CurrencySymbol, CurrencySymbolSize } from "@namehash/namekit-react";
+import { CurrencySymbol } from "@namehash/namekit-react/client";
```

Comment:
I'd rename client to something more elegant from external developers importing our lib.'s perpective, 

e.g. `client.ts` ---> `index.ts` while `index.ts` ---> `server-components.ts`

Which would result in:

`import { CurrencySymbol } from "@namehash/namekit-react";`
`import { Button, ButtonProps, Input } from "@namehash/namekit-react/server-components";`

Reply:
Open to suggestions on what we do here in the short term to finally make it so that we can successfully merge PR 342 without issues.

Please keep this simple for now and create separate shortcut stories as you think might be helpful for the future. Thank you.